src/styles/global.css

```
@@ -0,0 +1,157 @@
+
+/* ── Signal dot ──────────────────────────────── */
+.sdot {
+  width: 7px; height: 7px;
+  border-radius: 50%;
+  flex-shrink: 0;
+}
+.sdot--pulse {
+  animation: pulse-dot 2.4s ease-in-out infinite;
+}
+@keyframes pulse-dot {
+  0%, 100% { opacity: 1;   box-shadow: 0 0 0 0 currentColor; }
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)